pkg/apis/config/v1alpha1/resmgr.go

```diff
@@ -41,4 +43,8 @@ type CommonConfig struct {
 	Log log.Config `json:"log,omitempty"`
 	// +optional
 	Instrumentation instrumentation.Config `json:"instrumentation,omitempty"`
+	// +optional
+	RdtClasses rdt.Config `json:"rdt,omitempty"`
```

Comment:
We now have both mixed-case `Rdt` and all caps ` RDT` 

```suggestion
	RDTClasses rdt.Config `json:"rdt,omitempty"`
```

Reply:
Fixed to use everywhere RDT, where we can.